views/actions/session.jsx

```diff
@@ -1,27 +1,30 @@
-import { push } from 'react-router-redux';
-import { query } from '../helpers';
+import {push} from 'react-router-redux';
+import {query} from '../helpers';
 
 export const PROCESS_USER = 'PROCESS_USER';
 export const SIGNED_IN = 'SIGNED_IN';
 export const SIGNED_OUT = 'SIGNED_OUT';
+export const ERROR = 'AUTH_ERROR';
+export const RESET_AUTH_ERROR = 'RESET_AUTH_ERROR';
 
 function processUser() {
-  return {
```

Comment:
Do you think you can revert the beautify changes or change your beautify settings? It is causing a lot of false positives in the code change